src/test/java/uk/gov/pay/directdebit/common/clients/GoCardlessClientFacadeTest.java

```diff
@@ -37,32 +40,35 @@
 public class GoCardlessClientFacadeTest {
 
     private static final String BANK_NAME = "Awesome Bank";
-
-    private final BankAccountDetails bankAccountDetails = new BankAccountDetails(AccountNumber.of("12345678"), SortCode.of("123456"));
+    private static final MandateBankStatementReference BANK_STATEMENT_REFERENCE = MandateBankStatementReference.valueOf("BANK STATEMENT REF");
+    private static final GoCardlessMandateId GO_CARDLESS_MANDATE_ID = GoCardlessMandateId.valueOf("MD123");
+    private static final GoCardlessPaymentId GO_CARDLESS_PAYMENT_ID = GoCardlessPaymentId.valueOf("PM123");
+    private static final BankAccountDetails BANK_ACCOUNT_DETAILS = new BankAccountDetails(AccountNumber.of("12345678"), SortCode.of("123456"));
+    private static final LocalDate CHARGE_DATE = LocalDate.of(1969, JULY, 16);
 
     @Mock
     private GoCardlessClientWrapper mockGoCardlessClientWrapper;
 
     @Mock
-    private BankDetailsLookup mockBankDetailsLookup;
+    private com.gocardless.resources.BankDetailsLookup mockBankDetailsLookup;
```

Comment:
Is there a reason for not importing this `com.gocardless.resources` package?

Reply:
The reason is because we have some types like `Mandate` that exist in both our model domain and the GoCardless library. Using explicit fully-qualified types consistently in the field declarations makes it clear which is which.